kubernetes/yamls/development/api-server/api-deployment.yml

```diff
@@ -18,13 +18,8 @@ spec:
           httpGet:
             path: /api/status.json
             port: 80
-          initialDelaySeconds: 30
-          timeoutSeconds: 3
-        readinessProbe:
-          httpGet:
-            path: /api/status.json
-            port: 80
-          initialDelaySeconds: 30
+          initialDelaySeconds: 90
+          periodSeconds: 300
           timeoutSeconds: 3
```

Comment:
Well, I have tried with all possibilities and things are working fine. You can create the deployments one by one and check for their working - 
```bash
kubectl create -R -f kubernetes/yamls/development/elasticsearch
# Wait for elasticsearch to get ready
kubectl create -R -f kubernetes/yamls/development/api-server
```